pkg/apply/task/inv_set_task_test.go

```diff
@@ -160,6 +162,16 @@ func TestInvSetTask(t *testing.T) {
 			invalidObjs:   object.ObjMetadataSet{idInvalid},
 			expectedObjs:  object.ObjMetadataSet{id3},
 		},
+		"object failed to reconcile": {
```

Comment:
can you add permutations for both apply and delete reconcile failures?

Reply:
done